captum/insights/features.py

```diff
@@ -93,6 +123,12 @@ def visualize(self, attribution, data, contribution_frac) -> FeatureOutput:
 
 
 class TextFeature(BaseFeature):
+    r"""
+    TextFeature is used to visualize text (e.g. sentences) in Insights.
+    It expects the visualization transform to convert the input data (e.g. index to
+    string) to the raw text.
```

Comment:
This visualization transform hint is useful, might be good to add more particular descriptions / docstrings below for text.